data/schemas/edges/variants_genes.VariantEFFECTSAdapter.json

```diff
@@ -1,142 +1,146 @@
-    },
-    "label": {
-      "type": "string",
-      "description": "Label for the variant to gene edge",
-      "example": "variant effect on gene expression of PPIF"
-    },
-    "name": {
-      "type": "string",
-      "description": "Name for the variant to gene edge",
-      "enum": [
-        "modulates expression of"
-      ]
-    },
-    "inverse_name": {
-      "type": "string",
-      "description": "Inverse name for the variant to gene edge",
-      "enum": [
-        "expression modulated by"
-      ]
-    },
-    "source": {
-      "type": "string",
-      "description": "Source for the variant to gene edge",
-      "enum": [
-        "IGVF"
-      ]
-    },
-    "source_url": {
-      "type": "string",
-      "description": "Source URL for the variant to gene edge",
-      "example": "https://data.igvf.org/tabular-files/IGVFFI0524YUIL/"
-    },
-    "files_filesets": {
-      "type": "string",
-      "description": "Filesets for the variant to gene edge",
-      "example": "files_filesets/IGVFFI0332UGDD",
-      "pattern": "^files_filesets/"
-    },
-    "method": {
-      "type": "string",
-      "description": "Method for the variant to gene edge",
-      "example": "Variant-EFFECTS"
-    },
-    "simple_sample_summaries": {
-      "type": "array",
-      "description": "Simple sample summaries for the variant to gene edge",
-      "items": {
-        "type": "string"
+  "allOf": [
+    {
+      "$ref": "edges.base.json"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "_key": {
+          "type": "string",
+          "description": "Unique identifier for the variant to gene edge"
+        },
+        "_from": {
+          "example": "variants/NC_000001.11:10000:T:A",
+          "pattern": "^variants/",
+          "collections": [
+            "variants"
+          ]
+        },
+        "_to": {
+          "example": "genes/ENSG00000187583.10",
+          "pattern": "^genes/ENSG[0-9]{11}(?:_PAR_Y)?$",
+          "collections": [
+            "genes"
+          ]
+        },
+        "effect_size": {
+          "type": "number",
+          "description": "Effect size for the variant to gene edge"
+        },
+        "log2_fold_change": {
+          "type": "number",
+          "description": "Log2 fold change for the variant to gene edge"
+        },
+        "p_nominal_nlog10": {
+          "type": "number",
+          "description": "P nominal nlog10 for the variant to gene edge"
+        },
+        "fdr_nlog10": {
+          "type": "number",
+          "description": "FDR nlog10 for the variant to gene edge"
+        },
+        "power": {
+          "type": [
+            "number",
+            "null"
+          ],
+          "description": "Power for the variant to gene edge"
+        },
+        "class": {
+          "enum": [
+            "observed data"
+          ]
+        },
+        "label": {
+          "description": "Label for the variant to gene edge",
```

Comment:
you can remove all the description for the fields that are already defined in edges base schema